.github/workflows/ci.yml

```diff
@@ -16,8 +16,10 @@ jobs:
       - name: Start containers
         run: docker compose up -d
       - name: Run smoketests
-        # TODO: Fix flaky tests https://github.com/clockworklabs/SpacetimeDB/issues/630
         run: test/run-smoke-tests.sh --parallel -x bitcraftmini-pretest zz_docker-restart-module zz_docker-restart-repeating-reducer zz_docker-restart-sql
+      # These cannot run in parallel, obviously
+      - name: Run restarting smoketests
+        run: test/run-smoke-tests.sh zz_docker-restart-module zz_docker-restart-repeating-reducer zz_docker-restart-sql
```

Comment:
Were parallel restarts the main issue?

Reply:
No, I just fixed that as a drive-by

Reply:
Well, turns out turns out we need to run them as a separate task, for whatever reason.